WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -210,7 +210,8 @@ public static boolean isBlockEditorDefaultForNewPost(SiteModel site) {
             // Default to block editor when mobile editor setting is empty
             return true;
         } else {
-            return site.getMobileEditor().equals(SiteUtils.GB_EDITOR_NAME);
+            boolean isSimpleWPCom = site.isWPCom() && !site.isWPComAtomic();
```

Comment:
Not a blocking issue, but what would you think about instead extracting out something along these lines:

```
public static boolean alwaysDefaultToGutenberg(SiteModel site) {
  return site.isWPCom() && !site.isWPComAtomic();
}
```

and then calling that method both here and at `SiteSettingsFragment::1005`? My thinking is that we always want those two checks to be the same (anytime the editor toggle is unavailable we want to also want to force the default editor to be Gutenberg), and having the logic in a shared method makes it more likely they'll stay in sync.

Reply:
Nice suggestion, added here: 8d7b3dd28a78751f7698ca2dbb031df31ae2c295